src/scss/grommet-core/_objects.header.scss

```diff
@@ -5,6 +5,10 @@
   width: 100%;
   margin-bottom: 0px;
 
+  // Height is needed to coerce IE11 into aligning center even though
+  // this is ignored by min-height. (Also needed on header--small.)
+  height: $header-height - 1;
```

Comment:
Yeah, by setting it to something < the `min-height` it allows `min-height` to override it. If it's >= then the header will not be dynamic for longer text.

Also, -1 because I think that IE11 calculates the center based on the set height so we'd want that to be as close to actual center as possible.

It does see hackish but.... If you have any other suggestion for the work-around I'm all ears!